app/assets/javascripts/markdown_editor.js

```diff
@@ -72,7 +72,7 @@ const initializeMarkdownEditors = () => {
         change: () => {
           // Keep real form <textarea> in sync
           const content = toastEditor.getMarkdown();
-          formInput.value = content;
+          $(formInput).val(content).trigger('change');
```

Comment:
Since this file is copied over and is (so far) not using jQuery, I would prefer a vanilla JS solution:

```suggestion
          formInput.value = content;
          const event = new Event('change', { bubbles: true });
          formInput.dispatchEvent(event);
```